frontend/ui/widget/doublespinwidget.lua

```diff
@@ -117,6 +123,13 @@ function DoubleSpinWidget:update(numberpicker_left_value, numberpicker_right_val
     right_widget.picker_updated_callback = function(value)
         self:update(left_widget:getValue(), value)
     end
+    local separator_widget = TextBoxWidget:new{
+        text = self.is_range and "–" or "",
+        alignment = "center",
+        face = self.title_face,
+        bold = true,
+        width = math.floor(math.min(self.screen_width, self.screen_height) * 0.02),
```

Comment:
Feels a bit odd to have to use a TextBoxWidget, when everything else use a simpler TextWidget - but I guess you used that for the fixed width and centering. This fixed width computation feels also odd: we would probably be better with a TextWidget + some padding.
It's some UI finicky/tricky work you may not (yet) be confortable with, so I'll let this one pass :)